cedar-policy-validator/src/cedar_schema/err.rs

```diff
@@ -581,6 +581,7 @@ impl Diagnostic for MultipleEAMapDeclarationsError {
 
 /// Errors relating to embedded attribute maps (`EAMap`s)
 #[derive(Debug, Clone, Diagnostic, Error, PartialEq, Eq)]
+#[non_exhaustive]
```

Comment:
Assuming you're adding `non_exhaustive` to this error type (and `CedarSchemaParseErrors`) because it is publicly exposed -- can you add a CAUTION comment noting this?

Reply:
Actually, turns out it is not publicly exposed.  So it doesn't matter as much if we (do or don't) add `non_exhaustive`. Preference?

Reply:
Ok, it's fine as-is then. Just wanted to make sure we hadn't forgot to mark something as public 🙂

Reply:
`CedarSchemaParseErrors` is not publicly exposed either

Reply:
Added comments to that effect, at a minimum